api/opds2/publication.py

```diff
@@ -242,6 +250,23 @@ def setPreferredLink(self, editions):
             'type': firstLink.media_type,
             'rel': 'http://opds-spec.org/acquisition/open-access'
         })
+    
+    def setLinkIdentifier(self, link):
```

Comment:
one nit on the name of this method: I'd normally expect a method starting with `set` to mutate `self` in some way, but this just returns a value. How would you feel about renaming this to `getLinkIdentifier` or `determineLinkIdentifier` (or anything that indicates that this isn't doing a mutation)?

Reply:
Makes sense. I'll change the method to `getLinkidentifier`